### PR TITLE
Bugfix/fix mead

### PR DESCRIPTION
**Pull request by limebell**

### CODE REVIEW

File: .Lib9c.Tests/TempTest.cs [hunk 1]
```diff
@@ -0,0 +1,72 @@
...
+            Log.Debug("ValidatorAddress: {validatorAddress}", validatorAddress);
+            Assert.True(true);
+        }
+
+        // Test that loads a chain from local file.
+        [Fact]
+        public void LoadChainFromLocalFile()
+        {
+            // Arrange
+            var storePath = "C:\\Users\\lime_\\planetarium\\local-test\\mainnet-snapshot";
```
Comment (boscohyun) [inline .Lib9c.Tests/TempTest.cs:46]: Presumably this `TempTest` is not intended to be merged, right?

File: Lib9c/Action/RewardGold.cs [hunk 2]
```diff
@@ -38,7 +38,7 @@ public override void LoadPlainValue(IValue plainValue)
 
         public override IWorld Execute(IActionContext context)
         {
-            context.UseGas(1);
```
Comment (boscohyun) [inline Lib9c/Action/RewardGold.cs:41]: Is `RewardGold` action no longer gassing?

Comment (limebell) [reply Lib9c/Action/RewardGold.cs:41]: Despite of the line, it was treated as a `BlockAction`, so it did not consume gas actually at the libplanet level before. With changes in how actions are processed in libplanet, keeping that line could potentially cause issues, which is why it has been removed to maintain consistency with previous behavior.

Comment (boscohyun) [reply Lib9c/Action/RewardGold.cs:41]: Thanks for the explanation!

**Pull request merged**